website/static/js/contribAdder.js

```diff
@@ -59,6 +60,7 @@ var AddContributorViewModel = oop.extend(Paginator, {
         self.results = ko.observableArray([]);
         self.selection = ko.observableArray();
         self.notification = ko.observable('');
+        self.url = ko.observable('');
```

Comment:
This is in order to pass in different url for ajax request based on different search options.